packages/frontend/src/features/sqlRunner/store/barChartSlice.ts

```diff
@@ -195,6 +197,14 @@ export const barChartConfigSlice = createSlice({
             if (!state.config.fieldConfig) return;
 
             state.config.fieldConfig.y.splice(action.payload, 1);
+
+            // Remove series styling if only one y axis field left
+            if (
+                state.config.fieldConfig.y.length === 1 &&
+                state.config.display?.series
+            ) {
+                state.config.display.series = undefined;
+            }
```

Comment:
does this make sense to you @owlas?
So when there's only one y axis, then there's no extra fields, so we can clear the series styling. LMK if you want this somewhere else